Monika After Story/game/script-farewells.rpy

```diff
@@ -345,6 +345,51 @@ label bye_prompt_sleep:
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_SLEEP
     return 'quit'
 
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_to_games",
+            unlocked=True,
+            prompt="I'll play other games.",
+            pool=True
+        ),
+        eventdb=evhand.farewell_database
+    )
+
+label bye_prompt_to_games:
+    m 1esa "Ok [player]!"
+    m 1eka "I wish I could come too and see your main character."
```

Comment:
While this line itself is fine, it doesn't account for the fact that not all games have characters and it would be advisable to make this compatible with many sorts of games.